### PR TITLE
adding hw1

### PR DESCRIPTION
**Pull request by erinleeny**
only ruby and java currently here

### CODE REVIEW

**Comment by erinleeny** [comment]
@rickyc

File: homework-1.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 item = ARGV[0]
 price = ARGV[1].to_f/1.0
```
Comment (rickyc) [inline homework-1.rb:2]: If `ARGV[1]` is equal to `1` and you type cast it to float. It turns to `1.0`. What is `1.0/1.0`? 😂. The `/1.0` is redundant.

**Review by rickyc — APPROVED** [review]
👍 You can merge these in for now. Overall looks fine.

**Comment by rickyc** [comment]
@erinl1 I found it! This is what you need :)

```swift
let twoDecimalPlaces = String(format: "%.2f", 10.426123)
```

`sprintf` in C++/C means string print format! 😂 Makes so much sense!!

**Comment by rickyc** [comment]
@erinl1 Merge this in :)

**Pull request merged**